src/Resources/config/listener.yml

```diff
@@ -5,10 +5,15 @@
 # Symfony route listener (defaults to 32).
 #
 # - 30: ContainerScopeListener
-# - 28: RefererIdListener
-# - 26: InitializeSystemListener
-# - 24: ToggleViewListener
-# - 22: OutputFromCacheListener
+# - 20: RefererIdListener
+# - 20: LocaleListener
+#
+# The priorites of the following listeners must be lower than the one of the
+# Symfony locale listener (defaults to 16).
+#
+# - 10: InitializeSystemListener
+# - 5:  ToggleViewListener
```

Comment:
This is wrong. The `ToggleViewListener` needs to come before the `OutputFromCacheListener`.